webview/src/plots/components/constants.ts

```diff
@@ -1,12 +1,12 @@
 import { Config, FontWeight } from 'vega'
-import { getThemeValue, ThemeProperty } from '../../util/styles'
 
-const foregroundColor = getThemeValue(ThemeProperty.FOREGROUND_COLOR)
+const foregroundColor = 'var(--vscode-editor-foreground)'
```

Comment:
`getThemeValue` causes an issue here where the color it sets does not get updated when the color theme changes, requiring a webview re-open. Switching to a plain CSS variable fixes that.